### PR TITLE
Added methods in driver class to switch to next tab and parent tab

### PR DESCRIPTION
**Pull request by MohitKumarTangar**
Created two methods in Driver.java class.
1. switchToNextTab - using this method, driver can switch to next tab. NoSuchElementException is also handled in case there is no next tab present.
2. switchToParentTab - using this method, driver can switch to parent tab.

### CODE REVIEW

File: src/main/java/com/znsio/e2e/tools/Driver.java [hunk 1]
```diff
@@ -420,4 +421,23 @@ public void scrollTillElementIntoView(By elementId) {
...
         ((JavascriptExecutor) driver).executeScript("arguments[0].scrollIntoView(true);", element);
     }
+    public void switchToNextTab() {
+        Iterator<String> iterator = driver.getWindowHandles().iterator();
+        try {
+            iterator.next();
+            driver.switchTo().window(iterator.next());
+        } catch (NoSuchElementException e) {
+            LOGGER.error("Unable to get next window handle.");
+            e.printStackTrace();
```
Comment (anandbagmar) [inline src/main/java/com/znsio/e2e/tools/Driver.java:431]: we should not swallow the exception. There should be an exception thrown here 
ex:

`throw new NoSuchElementException("Unable to get next window handle.", e);`

In such approch, `e.printStackTrace();` is also not required

File: src/main/java/com/znsio/e2e/tools/Driver.java [hunk 2]
```diff
@@ -420,4 +421,23 @@ public void scrollTillElementIntoView(By elementId) {
...
+        }
+    }
+
+    public void switchToParentTab() {
+        try {
+            driver.switchTo().window(driver.getWindowHandles().iterator().next());
+        }catch (NoSuchElementException e){
+            LOGGER.error("No previous tab found.");
+            e.printStackTrace();
```
Comment (anandbagmar) [inline src/main/java/com/znsio/e2e/tools/Driver.java:440]: see the above message

**Comment by MohitKumarTangar** [comment]
Updated for both the methods.

File: src/main/java/com/znsio/e2e/tools/Driver.java (hunk 1 above)
Comment (MohitKumarTangar) [reply src/main/java/com/znsio/e2e/tools/Driver.java]: updated

File: src/main/java/com/znsio/e2e/tools/Driver.java (hunk 2 above)
Comment (MohitKumarTangar) [reply src/main/java/com/znsio/e2e/tools/Driver.java]: updated

**Pull request merged**